### PR TITLE
592 implement card nav

### PR DESCRIPTION
**Pull request by dexguitar**

### CODE REVIEW

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardNavigationTests.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    @Test
+    public void linkClickTest() {
+        cardNavigation.nav.highlight();
+
+        cardNavigation.activeLink.click();
+        validateAlert(is(activeLinkAlert));
+
+        cardNavigation.jdiLink.click();
+        WindowsManager windowsManager = new WindowsManager();
+        windowsManager.switchToWindow(2);
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardNavigationTests.java]: switchToWindow is a static method, you don't need to create new WindowsManager()

Comment (mf-code) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardNavigationTests.java]: WindowsManager.switchToWindow(2)

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardNavigationTests.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        assertEquals(getUrl() , jdiLinkRef);
+        windowsManager.closeWindow();
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardNavigationTests.java]: the same

File: jdi-light/src/main/java/com/epam/jdi/light/elements/common/UIElement.java [hunk 3]
```diff
@@ -618,8 +618,16 @@ protected boolean selected() {
             cl.contains("selected") || getAttribute("checked").equals("true");
     }
     protected boolean enabled() {
-        List<String> cls = classes();
-        return cls.contains("active") || !hasAttribute("disabled") || getWebElement().isEnabled() && !cls.contains("disabled");
+        if (hasClass("active")) {
+            System.out.println("hasClass Called");
```
Comment (mf-code) [inline jdi-light/src/main/java/com/epam/jdi/light/elements/common/UIElement.java]: now we can remove it

File: jdi-light/src/test/java/mockitotests/EnabledElementMockTests.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    private static final boolean webElementEnabled = true;
+    private static final boolean isEnabled = true;
+
+    @DataProvider
+    public Object[][] combinations() {
+        return new Object[][]{
+                {active, disabledByClass, disabledByAttribute, webElementEnabled, isEnabled},
+                {active, disabledByClass, !disabledByAttribute, webElementEnabled, isEnabled},
+                {active, disabledByClass, !disabledByAttribute, !webElementEnabled, isEnabled},
+                {active, disabledByClass, disabledByAttribute, !webElementEnabled, isEnabled},
```
Comment (mf-code) [inline jdi-light/src/test/java/mockitotests/EnabledElementMockTests.java]: please sort string as here:
                {active, disabledByClass, disabledByAttribute, webElementEnabled, isEnabled},
                {active, disabledByClass, disabledByAttribute, !webElementEnabled, isEnabled},
                {active, disabledByClass, !disabledByAttribute, webElementEnabled, isEnabled},
                {active, disabledByClass, !disabledByAttribute, !webElementEnabled, isEnabled},

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardNavigationTests.java (hunk 1 above)
Comment (dexguitar) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardNavigationTests.java]: Done

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardNavigationTests.java (hunk 2 above)
Comment (dexguitar) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardNavigationTests.java]: Done

File: jdi-light/src/main/java/com/epam/jdi/light/elements/common/UIElement.java (hunk 3 above)
Comment (dexguitar) [reply jdi-light/src/main/java/com/epam/jdi/light/elements/common/UIElement.java]: Done

File: jdi-light/src/test/java/mockitotests/EnabledElementMockTests.java (hunk 4 above)
Comment (dexguitar) [reply jdi-light/src/test/java/mockitotests/EnabledElementMockTests.java]: Done

**Comment by mf-code** [comment]
@RomanIovlev  we are ready, please review

File: jdi-light/pom.xml [hunk 5]
```diff
@@ -95,13 +95,13 @@
...
             <artifactId>mockito-all</artifactId>
-            <version>1.10.19</version>
+            <version>1.8.4</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
-            <version>6.14.3</version>
+            <version>7.0.0-beta1</version>
```
Comment (mf-code) [inline jdi-light/pom.xml]: it's not a good idea to use beta version

Comment (dexguitar) [reply jdi-light/pom.xml]: Reverted back to 6.14.3

**Pull request merged**